packages/nx/src/command-line/migrate.ts

```diff
@@ -777,11 +791,28 @@ async function generateMigrationsJsonAndUpdatePackageJson(
 ) {
   const pmc = getPackageManagerCommand();
   try {
+    let originalPackageJson = readJsonFile<PackageJson>(
+      join(root, 'package.json')
+    );
+    if (
+      ['nx', '@nrwl/workspace'].includes(opts.targetPackage) &&
+      (await isMigratingToNewMajor(
+        originalPackageJson.devDependencies?.['nx'] ??
+          originalPackageJson.devDependencies?.['@nrwl/workspace'],
+        opts.targetVersion
+      ))
+    ) {
+      await connectToNxCloudCommand(
```

Comment:
Maybe something like this instead:

We noticed you are migrating to a new major version, but are not taking advantage of Nx Cloud. Nx Cloud can make your CI up to 10 times faster. Learn more about it here: https://nx.app. Would you like to add it?

Reply:
Sure, sounds good. I'll change it out.